rpmlint/checks/TagsCheck.py

```diff
@@ -225,6 +228,15 @@ def check(self, pkg):
                         if prov not in (x[0] for x in pkg.provides()):
                             self.output.add_info('W', pkg, 'no-provides', prov)
 
+                if has_pc:
+                    found_pkg_config_dep = False
+                    for p in (x[0] for x in pkg.provides()):
+                        if p.startswith('pkgconfig('):
+                            found_pkg_config_dep = True
+                            break
+                    if not found_pkg_config_dep:
+                        self.output.add_info('W', pkg, 'no-pkg-config-provides')
```

Comment:
This should be fatal imo. I just looked on the report and we have no reports for it and when i triggered it manualy it is pretty fatal to lack the provides.

Reply:
It basically should never happen since a pkgconfig implementation should be in the buildroot in most distributions.